stories/templates/Drawer/basic.js

```diff
@@ -0,0 +1,29 @@
+import { html } from "lit-html";
+
+export const Template = args => {
+  return html`
+    <sgds-button @click=${showDrawer}>Open ${args.placement} Drawer</sgds-button>
+    <sgds-drawer
+      ?open=${args.open}
+      label=${args.label}
+      placement=${args.placement}
+      ?noHeader=${args.noHeader}
+      ?noAnimation=${args.noAnimation}
+    >
+      This is a Drawer
+      <sgds-button @click=${closeDrawer} slot="footer" variant="link" class="close-drawer">Close</sgds-button>
+      <sgds-button slot="footer" variant="primary" type="submit" form="formA">Submit</sgds-button>
+    </sgds-drawer>
+  `;
+};
+
+export const showDrawer = () => {
+  const drawer = document.querySelector("sgds-drawer");
+  drawer.show();
+};
+export const closeDrawer = () => {
+  const drawer = document.querySelector("sgds-drawer");
+  drawer.hide();
+};
+
```

Comment:
```suggestion
export const args = {};
```

Reply:
There should be some console errors on dev env if you dont add this line. 

I have set up a plop template for autogeneration of boiler plate code for new components so that we dont need to be bothered with the boilerplate code and setting up of files. 

Can try out next time:)  run```npm run write:component```

Reply:
Thanks for the enhancement!!